client/app/services/service-details/service-details.html

```diff
@@ -7,8 +7,24 @@
         <li class="active">{{vm.title}}</li>
       </ol>
   </span>
+  <span class="service-details-toolbar-buttons">
```

Comment:
I tweaked its positioning just a little.  I guess its just the spacing in the icon.